Source/WebCore/html/canvas/WebGLUtilities.h

```diff
@@ -53,14 +53,14 @@ class ScopedInspectorShaderProgramHighlight {
     void hideHighlight();
 
     struct {
-        GCGLfloat color[4];
-        GCGLenum equationRGB;
-        GCGLenum equationAlpha;
-        GCGLenum srcRGB;
-        GCGLenum dstRGB;
-        GCGLenum srcAlpha;
-        GCGLenum dstAlpha;
-        GCGLboolean enabled;
+        GCGLfloat color[4] { 0, 0, 0, 0 };
```

Comment:
The constructor and destructor ensure the values are initialized even without the additions.